docs/Using-Fleet/REST-API.md

```diff
@@ -5170,6 +5172,14 @@ Returns a list of all queries in the Fleet instance.
     "name": "query1",
     "description": "query",
     "query": "SELECT * FROM osquery_info",
+    "team_id": null,
+    "frequency": 3600,
+    "platform": null,
+    "min_osquery_version": null,
```

Comment:
What does `null` look like in the UI?

Reply:
See above.  Same exercise needs to be done for this.  I doubt it should be `null`.  It should probably be `""`, but it'll take testing that.

I also see it named `"version"` elsewhere.  Are we going to change that here?  We could change it just here.  Someone needs to own the vision of "ok now we need to update it for policies" and that person needs to own bringing that to prioritization.

Reply:
`"min_osquery_version": ""`